packages/plugin-stripe/src/webhooks/index.ts

```diff
@@ -21,6 +21,7 @@ export const handleWebhooks: StripeWebhookHandler = async (args) => {
   if (syncConfig) {
     switch (method) {
       case 'created': {
+        // eslint-disable-next-line @typescript-eslint/await-thenable
```

Comment:
I usually add `void` instead. I'm not sure what everyone else on the team is doing for this.

Reply:
Updated to use `void` instead